google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -468,6 +469,46 @@ def jobs all: nil, token: nil, max: nil, filter: nil
           Job::List.from_gapi gapi, service, all, max, filter
         end
 
+        ##
+        # Retrieves the list of all projects for which the currently authorized
+        # account has been granted any project role.
+        #
+        # @param [String] token A previously-returned page token representing
+        #   part of the larger set of results to view.
+        # @param [Integer] max Maximum number of projects to return.
+        #
+        # @return [Array<Google::Cloud::Bigquery::ProjectReference>] (See
+        #   {Google::Cloud::Bigquery::ProjectReference::List})
+        #
+        # @example
+        #   require "google/cloud"
+        #
+        #   gcloud = Google::Cloud.new
+        #   bigquery = gcloud.bigquery
+        #
+        #   projects = bigquery.projects
+        #   projects.each do |project|
+        #     puts project.project_id
+        #   end
+        #
+        # @example Retrieve all projects: (See {ProjectReference::List#all})
+        #   require "google/cloud"
+        #
+        #   gcloud = Google::Cloud.new
+        #   bigquery = gcloud.bigquery
+        #
+        #   projects = bigquery.projects
+        #   projects.all do |project|
+        #     puts project.project_id
+        #   end
+        #
+        def projects all: nil, token: nil, max: nil
```

Comment:
Why is `all` an optional named argument? Its not listed in the documentation. The value is sent to `list_projects`, but `list_projects` does not use it.